event.go

```diff
@@ -96,6 +121,81 @@ func formatHeaders(h http.Header, redact map[string]bool) map[string]string {
 	return ret
 }
 
+func redactValues(b any, includedKeys map[string]bool) any {
+	lowerCase := regexp.MustCompile("[a-z]")
+	upperCase := regexp.MustCompile("[A-Z]")
+	numeric := regexp.MustCompile("[0-9]")
+	special := regexp.MustCompile("[^a-zA-Z0-9\\s]")
+
+	var redactInt = func(i int) int {
+		s := strconv.Itoa(i)
+		s = numeric.ReplaceAllString(s, "1")
+		i, _ = strconv.Atoi(s)
+		return i
+	}
+
+	switch b := b.(type) {
+	case map[string]any:
+		for k, v := range b {
+			if includedKeys[k] {
+				b[k] = v
+			} else {
+				b[k] = redactValues(v, includedKeys)
+			}
+		}
+		return b
+	case []any:
+		for i, v := range b {
+			b[i] = redactValues(v, includedKeys)
+		}
+		return b
+	case []uint8:
+		return "binary"
+	case string:
+		b = upperCase.ReplaceAllString(b, "A")
+		b = lowerCase.ReplaceAllString(b, "a")
+		b = numeric.ReplaceAllString(b, "1")
+		b = special.ReplaceAllString(b, "*")
+		return b
+	case bool:
+		b = false
+		return b
+	case int:
+		b = redactInt(b)
+		return b
+	case float64:
+		if b == math.Trunc(b) {
+			b = float64(redactInt(int(b)))
+			return b
+		} else {
+			s := fmt.Sprintf("%f", b)
+			s = numeric.ReplaceAllString(s, "1")
+			b, err := strconv.ParseFloat(s, 64)
+			if err != nil {
+				b = 0
+			}
+			return b
+		}
+	case float32:
```

Comment:
When you parse JSON into a `map[string]any` it will only contain `float64`'s so you can probably remove the code for ints and float32s

Reply:
Oh yeah ! I had seen that in the JSON Demarshalling , forgot to change it.